src/navsections/K8sResourceSectionBlueprint/ResourceKindInformation.styled.tsx

```diff
@@ -0,0 +1,15 @@
+import {InfoCircleOutlined as RawInfoCircleOutlined} from '@ant-design/icons';
+
+import styled from 'styled-components';
+
+import Colors from '@styles/Colors';
+
+export const InfoCircleOutlined = styled(RawInfoCircleOutlined)<{isSelected: boolean}>`
+  ${props => {
+    if (props.isSelected) {
+      return `color: ${Colors.blackPure};`;
+    }
+    return `color: ${Colors.blue9};`;
+  }};
```

Comment:
Here you could simply write `color: ${ props => props.isSelected ? Colors.blackPure : Colors.blue9 }`